content/source/docs/enterprise/api/index.html.md

```diff
@@ -161,6 +161,33 @@ $ curl \
 }
 ```
 
+## Rate Limiting
+
+You can make up to 30 requests per second to the API as an authenticated or unauthenticated request. If you reach the rate limit then you will be throttled and an error response will be returned.
```

Comment:
Throttling a person means strangling them!! 😬 Maybe let's change that to "your access will be throttled". 😂